### ISSUE TITLE
SSR : how to render react component using ssr 

### ISSUE THREAD

**Issue by ansonla3** [kadirahq/flow-router#637]
I have installed the flow-router-ssr, but do not know to make the react component could do the server side rending. 

What I have done is so far. is having a react component in the server folder then import it in the client side routes.jsx, then use the react-mounter to mount it, but there is nothing show on the screen.

anyone could help ? Please help,

**Comment by sahanDissanayake** [comment]
Have a look at these samples

https://github.com/kadira-samples/meteor-data-and-react

**Comment by ansonla3** [comment]
Yes, I read before,

this only in the container set the title, meta description. I want to render the whole react component in html format so that google could crawl the web page easily. 

Do you have any ideas ? thanks very much .

**Comment by sahanDissanayake** [comment]
yep, the example link if gave you actually render the html on the server side. So google can crawl. Have you actually ran the code on your computer to see if it renders on the server or not ?

**Comment by ansonla3** [comment]
ya , it works with that project.

but when use this project https://github.com/mantrajs/kickstart-mantrajs-webpack , remove the existing flow-router and install the flow-router-ss, it didn't work as expected. not sure what is the reason. Do you think that is related to webpack or mantrajs ? Can you help to have a try ? 

Thanks very much.

**Comment by CharlesOy** [comment]
I have the same question, have you fixed that.

**Comment by lyquocnam** [comment]
have you import routes from meteor server code ? webpack do not load files auto.

ex:
routes.js
```
...
FlowRouter.route('/', {
    action() {
      mount(Layout);
   }
}
```

server.js
```
import 'both/routes';
```
